test/e2e/specs/published-content/reader__view.ts

```diff
@@ -18,28 +17,25 @@ declare const browser: Browser;
 describe( DataHelper.createSuiteTitle( 'Reader: View' ), function () {
 	let page: Page;
 	let readerPage: ReaderPage;
-	const user: TestAccountName =
+	const accountName: TestAccountName =
 		envVariables.JETPACK_TARGET === 'remote-site' ? 'jetpackRemoteSiteUser' : 'commentingUser';
-	const expectedSiteName =
-		envVariables.JETPACK_TARGET === 'remote-site'
-			? SecretsManager.secrets.testAccounts.jetpackRemoteSiteUser.testSites?.primary.url
-			: SecretsManager.secrets.otherTestSites.notifications;
 	let testAccount: TestAccount;
 
 	beforeAll( async function () {
 		page = await browser.newPage();
-		testAccount = new TestAccount( user );
+		testAccount = new TestAccount( accountName );
 		await testAccount.authenticate( page );
 	} );
 
-	it( 'View the Reader stream', async function () {
+	it( 'Visit the Reader', async function () {
 		readerPage = new ReaderPage( page );
 		await readerPage.visit();
-		const siteOfLatestPost = await readerPage.siteOfLatestPost();
-		expect( siteOfLatestPost ).toEqual( expectedSiteName );
 	} );
 
-	it( 'Visit latest post', async function () {
-		await readerPage.visitPost( { index: 1 } );
+	it( 'Reader stream is present', async function () {
+		await Promise.any( [
+			page.getByRole( 'link', { name: 'Find sites to follow' } ),
+			page.getByRole( 'article' ),
```

Comment:
**Suggestion (non-blocking):** Hmm, is there any way we can make this selector a bit more specific? Looking for _any_ `article` element on the page seems a bit risky and prone to false-negatives (like if there ended up being a separate sidebar with an article element in the future, this could always pass). I don't think we need to be too specific, but selecting on only an HTML element role seems a bit risky on a full page!

Reply:
> Looking for any article element on the page seems a bit risky and prone to false-negatives

Fair points. I had chosen to use the raw `article` role selector because this is the a11y tree of the Reader stream that's populated with articles:

![image](https://github.com/Automattic/wp-calypso/assets/6549265/22af47f1-66eb-49b7-87c4-8ec096e0dd26)

Pretty bad, I say. None of the intermediate elements have names, labels or such. 

My updated proposal is to anchor the `article` on the `main` role element, which should help prevent stray `article` role elements elsewhere from messing up our spec.

Reply:
Actually, let's zoom back on the question even more.

Is there really a need to switch between the user accounts here? Why can't the spec just use one test account regardless of the `JETPACK_TARGET` value?

https://github.com/Automattic/wp-calypso/blob/trunk/test/e2e/specs/published-content/reader__view.ts#L22

Reply:
I'm going to merge this change for now to unmute the failure.